site/package.json

```diff
@@ -40,6 +40,7 @@
     "react-dom": "^16.9.0",
     "react-helmet": "5.2.1",
     "react-media": "^1.10.0",
+    "sass": "^1.49.0",
```

Comment:
Upgrading to the latest version of the npm `sass` (which uses dart scss compiled to scss rather than "native" or "embedded" dart sass) is fine for compiling the Kaizen site because it's relatively small and the sass compile times aren't too bad.

However, If this PR is introducing a new scss feature that's not backwards compatible with the latest version of https://www.npmjs.com/package/node-sass (it stopped getting updates a few years ago), then this will cause a problem for our huge consuming codebases. Last time we tried to use dart scss (non embedded) in Big M for example, it took several minutes and the machine ran out of memory before it could even finish.

So the problem is that this would be blocking consuming repos from using these new versions of components until they have upgraded to dart-sass. Although a stable version of embedded/native dart sass was released very recently, I'm not sure that's everything - I think the webpack plugin also needs a release that has the option of using embedded sass (this may have already happened though). It just might not be that trivial and it's a bit unproven at this point.

On the other hand, it'd be a driving force for consumers to upgrade to dart sass. I'm not sure what's better or worse - I will have to leave your team to decide on that one, but I think it's worth consulting with the biggest consumers as to whether upgrading to dart-scss is something that's feasible for them.